src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Sulu\Bundle\CoreBundle\DependencyInjection;
 
+use Oro\ORM\Query\AST\Functions\Cast;
```

Comment:
should we mark the `Sulu\Component\Rest\DQL\Cast` as deprecated?

Reply:
Would definitely do this